cost/google/cud_recommendations/google_cud_recommendations.pt

```diff
@@ -0,0 +1,212 @@
+name "Google Committed Use Discount Recommender Policy"
+rs_pt_ver 20180301
+type "policy"
+short_description "This Policy finds Google CUD Recommendations and reports when it finds them. See the [README](https://github.com/flexera/policy_templates/tree/master/cost/google/recommender) and [docs.rightscale.com/policies](https://docs.rightscale.com/policies/) to learn more."
+category "Cost"
+severity "low"
+default_frequency "daily"
+info(
+  version: "2.0",
+  provider:"Google",
+  service: "Storage",
+  policy_set: "Native Recommendations"
+)
+
+###############################################################################
+# Parameters
+###############################################################################
+
+parameter "param_email" do
+  type "list"
+  label "Email addresses to notify"
+  description "Email addresses of the recipients you wish to notify when new incidents are created"
+end
+
+parameter "param_location_whitelist" do
+  type "list"
+  label "Location"
+  description "Location to check, it can be the zone, region or global"
```

Comment:
We might be able to remove this because cuds are only supported regionally. I think